aws-lambda-layer/Rakefile

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+BUILD_DIR = File.join(__dir__, 'build')
+
+ENV['FREETDS_VERSION'] ||= '1.1.6'
```

Comment:
Is 1.1.6 the latest version? Struggling any reference to FreeTDS editions online.

Reply:
1.1.40 is the latest release.  I defaulted to 1.1.6 as it was the latest at the time of writing.  I can add a link to where to find the latest release.  Although the way these tasks are written, we are pulling freetds from this repo since their ftp goes down all the time.

1:  https://www.freetds.org/software.html

Reply:
I definitely did not read "1.1.40" on the main site as "1.1.4" and think we were on some experimental release...nope would never do that...

![naw](https://media.giphy.com/media/dCixq2fQBT2xy/giphy.gif)

Reply:
I was confused for a solid minute before linking you the site as well :)